src/components/templates/FaqTemplate.js

```diff
@@ -1,10 +1,54 @@
-import React from 'react';
-import { Link } from 'gatsby';
+import React from "react";
+import { Link } from "gatsby";
+import Heading from "../atoms/Heading";
+import Fontain from "../atoms/Fontain";
+import FontainImg from "../../../static/assets/Fontain.png";
+import FaqPair from "../molecules/FaqPair";
+import styled from "@emotion/styled";
+
+const ContentContainer = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  margin: "0 29.17%"
+});
 
 const FaqTemplate = () => (
   <div>
-    <p>FAQ!</p>
     <Link to="/">Go Back</Link>
+    <ContentContainer>
+      <Heading>Časté otázky</Heading>
+      <Fontain src={FontainImg} alt="fontain" />
+      <FaqPair
```

Comment:
move the question and answers to a separate JSON file in `src/data/faq.json`
then just import and map the file, this way you will end up with a single rendering function and much less code :)